fhirpipe/api/routes.py

```diff
@@ -0,0 +1,49 @@
+from flask import Blueprint, request, jsonify
+
+import logging
+
+from fhirpipe.errors import OperationOutcome
+from fhirpipe.cli.run import run as fp_run
+from fhirpipe.extract.graphql import get_credentials
+from fhirpipe.extract.sql import get_connection
+
+
+api = Blueprint("api", __name__)
+
+
+@api.route("/run", methods=["POST"])
+def run():
+    default_params = {
+        "mapping": None,
+        "source": None,
+        "resources": None,
+        "reset_store": False,
+        "chunksize": None,
+        "bypass_validation": False,
+        "multiprocessing": False,
+    }
+
+    body = request.get_json()
+
+    # Merge body with default_params to get parameters to use
+    params = {k: body[k] if k in body else default_params[k] for k in default_params}
```

Comment:
The problem is that I also wanted to send the credential_id in the request body and if I use `{**default_params, **body}`, it will be given to `run()` and will result in an unexpected argument error

Reply:
:+1: